docs/source/technical_tutorials/access/accessing_LVIS_data.ipynb

```diff
@@ -0,0 +1,1765 @@
+{
```

Comment:
<p>Could probably clear output of this cell, trying to remember the "fix" for this. Was it creating a new workspace?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>